apps/cowswap-frontend/src/modules/trade/const/tradeUrl.ts

```diff
@@ -1,2 +1,3 @@
 export const TRADE_URL_SELL_AMOUNT_KEY = 'sellAmount'
 export const TRADE_URL_BUY_AMOUNT_KEY = 'buyAmount'
+export const TRADE_URL_ORDER_KIND_KEY = 'orderKind'
```

Comment:
Can we make it shorter?

```suggestion
export const TRADE_URL_ORDER_KIND_KEY = 'kind'
```

Reply:
But why? `kind` sounds too generic

Reply:
Just because I like it shorter.
OrderKind feels too verbose.
But I'm fine if you want to keep it.